loadtest.go

```diff
@@ -141,21 +254,38 @@ func (l *loadTest) runPublisher(ctx context.Context, client Client, userNum int6
 
 	l.log.Debug("starting publisher", "channels", channels, "interval", l.w.Conf().Publisher.PublishInterval)
 
+	pushDeviceConfig := l.w.Conf().Publisher.PushDevice
 	errG, ctx := errgroup.WithContext(ctx)
 	for i := range channels {
 		channel := channels[i]
+		if pushDeviceConfig.Enabled {
+			channel = pushDeviceConfig.Namespace + ":" + channel
```

Comment:
Rather than having this conditional in the code, can't the publisher just be configured with the namespaced channel? i.e.:

```yaml
publisher.channels: "pushenabled:fanout"
```